app/views/admin_views.py

```
@@ -613,3 +1033,525 @@ def post(self):
 
         return jsonify({"message": "Training template created successfully", "training_id": training.training_id}), 201
 
+from flask import request, jsonify
```

Ruff (0.12.2)

Redefinition of unused `request` from line 959

Remove definition: `request`

(F811)